addon/components/discover-page/component.js

```diff
@@ -146,110 +116,33 @@ export default Ember.Component.extend(Analytics, hostAppName, {
      * @property {Object} filterReplace
      */
     filterReplace: {},
-    /**
-     * Funders query parameter.  If "funders" is one of your query params, it must be passed to the component so it can be reflected in the URL.
-     * @property {String} funders
-     * @default ''
-     */
-    funders: '',
-    /**
-     * Institutions query parameter.  If "institutions" is one of your query params, it must be passed to the component so it can be reflected in the URL.
-     * @property {String} institutions
-     * @default ''
-     */
-    institutions: '',
 
-    /**
-     * Language query parameter.  If "language" is one of your query params, it must be passed to the component so it can be reflected in the URL.
-     * @property {String} language
-     * @default ''
-     */
-    language: '',
-    loading: true,
     /**
      * Locked portions of search query that user cannot change.  Example: {'sources': 'PubMed Central'} will make PMC a locked source.
      * @property {Object} lockedParams
      */
     lockedParams: {},
-    numberOfResults: 0,  // Number of search results returned
-    numberOfSources: 0, // Number of sources
-    /**
-     * Organizations query parameter.  If "organizations" is one of your query params, it must be passed to the component so it can be reflected in the URL.
-     * @property {String} organizations
-     * @default ''
-     */
-    organizations: '',
-    /**
-     * Page query parameter.  If "page" is one of your query params, it must be passed to the component so it can be reflected in the URL.
-     * @property {Integer} page
-     * @default 1
-     */
-    page: 1,
-    /**
-     * Provider query parameter.  If "provider" is one of your query params, it must be passed to the component so it can be reflected in the URL.
-     * @property {String} provider
-     * @default ''
-     */
-    provider: '',
-    /**
-     * Publishers query parameter.  If "publishers" is one of your query params, it must be passed to the component so it can be reflected in the URL.
-     * @property {String} publishers
-     * @default ''
-     */
-    publishers: '',
-    /**
-     * q query parameter.  If "q" is one of your query params, it must be passed to the component so it can be reflected in the URL.
-     * @property {String} q
-     * @default ''
-     */
-    q: '',
```

Comment:
Could leave some of these defined

Reply:
Yeah, isn't `q` used below?

Reply:
`q` is passed in and managed on the controller. Leaving it defined on the component would just make it more clear what all is passed in. The downside is having a bunch of properties defined that are actually defined elsewhere.

Reply:
That's fine then, wasn't sure if it was always passed or just sometimes (i.e., if it would break anything if not defined). Leave as is then I say.

Reply:
Went ahead and added definitions for the ones that should always be passed in.